main.tex

```diff
@@ -39,7 +54,57 @@ \section{Introduction}
 
 The use of standard denominations in the resulting CoinJoin transaction obscures the relationship between individual inputs and outputs, making the origins of each output ambiguous. Unfortunately standard denominations also present limitations, such as requiring change outputs from these CoinJoin transactions, and limiting the usefulness of the privacy enhanced outputs for payments of arbitrary amounts.
 
-\section{Our Contribution}
+\subsection{Limitations of ZeroLink}
+In this work, we are aiming to improve ZeroLink, the most popular Chaumian CoinJoin implementation for Bitcoin. We identify several privacy shortcomings and inefficiencies of the current protocol as implemented by Wasabi. Some metrics comparing Wasabi, Samourai and other apparent CoinJoin transactions are provided. ``Other'' includes JoinMarket, but also has an inherent false positive error given these transactions are identified heuristically.
+
+\subsubsection{Round denominations}
+
+Due to the nature of blind signatures, mixed outputs of ZeroLink CoinJoins are restricted to a fixed set of value denominations which are multiples of a base denomination\footnote{approximately $0.1$\bitcoinA{}}.
```

Comment:
This doesn't follow. What is the nature of blind signatures that makes zerolink denominations fixed?

Reply:
because a blind signature by a key can only convey 1 bit of information from the signer to the verifier

Reply:
I don't think it matters, but here you go my nitpicking:

- ZeroLink only has a single denomination.
- Wasabi has multiple denomination, which are the multiples of each other, but they don't have to be.

Reply:
i think we should rename the whole section to Limitations of Wasabi, and just say that wasabi implements a version of ZeroLink, i think that would be more accurate, and clearer for readers who vaguely know have heard of wasabi but don't know the details